main.go

```diff
@@ -109,24 +110,30 @@ func main() {
 				amount       float64
 				secondAmount float64
 				delta        float64
+				deltaPercent float64
 			}
 
 			var finalResultsCosts []ServiceCosts
 			finalResultsCosts = make([]ServiceCosts, 0, len(allServiceNames))
 			var totalAmount = 0.0
 			var totalSecondAmount = 0.0
 			var totalDelta = 0.0
+			var totalDeltaPercent = 0.0
 			for _, service := range allServiceNames {
 				amount, _ := firstResultsCosts[service]
 				secondAmount, _ := secondResultsCosts[service]
 				secondAmount *= multiplier
 				delta := secondAmount - amount
+				deltaPercent := delta / amount * 100
```

Comment:
I think there's a few things here:

1. We can assume that if the delta is `0` we can just set the percentage to `0.0%`, but it outputs that anyway.
2. If the `amount` is `0`, then this get a `NaN` since you can't divide by 0. So, I think what we can do there is either output like the ∞ symbol or basically do some sort of calculation that turns the delta into a percentage based on the new amount. So, going from `0` to `5` is like 500%. But, I'm not sure I'm in favor of the latter. I just think the some output that isn't `NaN` is likely more helpful.

Reply:
@rgoomar when amount is `0` we get `inf` (not `NaN`) which is kind of what you mentioned - seems ok to me? Those also show up at the top when sorting by deltapercent:

```
+-------------------------------------------------+-------------+-------------+------------+---------------+
| SERVICE                                         | 2020-09-01  | 2020-10-01  | DELTA      | DELTA PERCENT |
+-------------------------------------------------+-------------+-------------+------------+---------------+
| Amazon Macie                                    |       $0.00 |      $56.40 |     $56.40 |        +Inf% |
| Amazon Comprehend                               |       $0.00 |       $0.02 |      $0.02 |        +Inf% |
| Amazon EC2 Container Service                    |      $64.22 |     $151.43 |     $87.21 |       135.8% |
| CodeBuild                                       |      $32.01 |      $50.62 |     $18.61 |        58.1% |
| AWS Config                                      |   $1,508.67 |   $2,326.21 |    $817.54 |        54.2% |
| Amazon Inspector                                |      $33.99 |      $47.79 |     $13.80 |        40.6% |```